ebu_tt_live/bindings/validation/timing.py

```diff
@@ -196,6 +196,12 @@ def _semantic_postprocess_timing(self, dataset, element_content):
                     # Propagate the longest end time among the children
                     self._computed_end_time = max(children_computed_end_times)
 
+        if begin_timedelta is not None \
+        and end_timedelta is not None \
+        and begin_timedelta > end_timedelta:
```

Comment:
I think the test is actually `>=` since begin time expressions are inclusive and end ones are exclusive, i.e. the presentation must have stopped by the end time.